mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_backup_policy.go

```diff
@@ -235,10 +236,10 @@ func resourceMongoDBAtlasCloudProviderSnapshotBackupPolicyImportState(d *schema.
 
 func snapshotScheduleUpdate(d *schema.ResourceData, conn *matlas.Client, projectID, clusterName string) error {
 	req := &matlas.CloudProviderSnapshotBackupPolicy{
-		ReferenceHourOfDay:    d.Get("reference_hour_of_day").(int),
-		ReferenceMinuteOfHour: d.Get("reference_minute_of_hour").(int),
-		RestoreWindowDays:     d.Get("restore_window_days").(int),
-		UpdateSnapshots:       cast.ToBool(d.Get("update_snapshots").(bool)),
+		ReferenceHourOfDay:    pointy.Int64(cast.ToInt64(d.Get("reference_hour_of_day"))),
+		ReferenceMinuteOfHour: pointy.Int64(cast.ToInt64(d.Get("reference_minute_of_hour"))),
+		RestoreWindowDays:     pointy.Int64(cast.ToInt64(d.Get("restore_window_days"))),
+		UpdateSnapshots:       pointy.Bool(d.Get("update_snapshots").(bool)),
```

Comment:
Isn't this from another PR?  Not sure why this is in this one?

Reply:
Oh is this the fix for PR #230 ?

Reply:
Yes, there are another PR with this fix, but I changed because it gives an error about wrong types when I'm trying to run tests about pagination, I can comment those line if you want and I didn't test since there's already another PR made by gonzalo.

Reply:
That makes sense - if you branched this from master PR #230 is  not merged so you didn't have the fix.   I would comment it out if tests are okay okay now, have @gmlp merge #230 before you merge this - sound about right?